### ISSUE TITLE
Apple Home Key

### ISSUE THREAD

**Issue by OrigamiDream** [homebridge/HAP-NodeJS#948]
### Current Situation

Apple Home Key has been introduced from WWDC21.
However it is still unavailable in HAP-NodeJS despite of it's 2022.

### Proposed Change

Although it is currently only available for MFi-certified accessories, there are possibilities that the spec would be revealed via data-mining like the case of HomeKit Secure Video.
If this happens, Homebridge owners can develop their own custom Home Key in their Wallet app.

**Comment by sabaatworld** [comment]
Would love to see support for this. Is it possible to implement or is it locked down by Apple somehow?

**Comment by Supereg** [comment]
> Although it is currently only available for MFi-certified accessories, there are possibilities that the spec would be revealed via data-mining like the case of HomeKit Secure Video.
If this happens, Homebridge owners can develop their own custom Home Key in their Wallet app.

Well the key difference here between Home Key and HKSV video is, that there were wildly available devices shipping with HKSV which could be used to reverse the protocol. Also Home Key will (probably?) require more expertise as one needs to analyze the NFC traffic between the accessory and the NFC device (at least I'm not an expert with NFC communications and don't have any equipment for it).

> Is it possible to implement or is it locked down by Apple somehow?

It is not locked down (afaik).

To sum up, yes it would be nice if we had support for this. But don't expect support to arrive anytime near. AFAIK I don't know anyone working on it right now. Also I wouldn't expect wide adoption. The amount of people or devices which could take advantage of this wouldn't be that big (after all we require a NFC reader).
